epsagon/lambda_wrapper.go

```diff
@@ -60,14 +70,12 @@ func (handler *epsagonLambdaWrapper) preInvokeOps(
 			info = &preInvokeData{
 				LambdaContext:      lc,
 				StartTime:          startTime,
-				InvocationMetadata: map[string]string{},
+				InvocationMetadata: metadata,
```

Comment:
I don't think it matters, because the function defined here will take the values that it sees when it is defined. I think that updating `metadata` afterwards will not update it here.